src/common/main/kotlin/tech/thatgravyboat/skyblockapi/utils/text/Text.kt

```diff
@@ -147,6 +148,18 @@ object TextUtils {
 
 }
 
+@Stub
+internal expect fun MutableComponent.withFont(location: ResourceLocation?): MutableComponent
+
+@Stub
+internal expect fun MutableComponent.font(): ResourceLocation?
+
+@Stub
+internal expect fun MutableComponent.withFontDescriptor(location: FontDescriptor?): MutableComponent
```

Comment:
Idk how I feel about this because if you try and use anything but normal font on < 1.21.9 it'll do nothing